src/handle-data.js

```diff
@@ -15,12 +15,20 @@ export function getColumnOrder({ qColumnOrder, qDimensionInfo, qMeasureInfo }) {
   return qColumnOrder?.length === columnsLength ? qColumnOrder : [...Array(columnsLength).keys()];
 }
 
-export function getColumnInfo(qHyperCube, colIndex) {
+export function getColumnInfo(qHyperCube, colIndex, layout) {
   const { qDimensionInfo, qMeasureInfo } = qHyperCube;
   const numDims = qDimensionInfo.length;
   const isDim = colIndex < numDims;
   const info = isDim ? qDimensionInfo[colIndex] : qMeasureInfo[colIndex - numDims];
   const isHidden = info.qError?.qErrorCode === 7005;
+  const totalInfo = isDim
```

Comment:
you need tests for this part as well

Reply:
great :)

Reply:
I'm not fan of them either... but why not ?

Reply:
The moment you start nesting them it becomes very hard to read.